packages/jsii-pacmak/lib/util.ts

```diff
@@ -69,14 +69,23 @@ export async function shell(
           return ok(out);
         }
         const err = Buffer.concat(stderr).toString('utf-8');
-        if (code != null) {
-          return ko(
-            new Error(`Process exited with status ${code}\n${out}\n${err}`),
-          );
-        }
         return ko(
-          new Error(`Process terminated by signal ${signal}\n${out}\n${err}`),
+          new Error(
+            `Process exited with ${
```

Comment:
nit: I still like 'terminated' over 'exited' to make it clear it was unexpected.

Reply:
I've actually seen "failed" used, which I like a little more than "terminated" (because termination - to me - is pertaining to SIGKILL / SIGTERM. But that is a jolly good suggestion!